### ISSUE TITLE
Login error: Failed to obtain SequenceID

### ISSUE THREAD

**Issue by pv** [jgeboski/bitlbee-facebook#36]
Fails to log in, as follows:
```
account facebook on
<@root> facebook - Logging in: Connecting
<@root> facebook - Logging in: Requesting authentication token
<@root> facebook - Logging in: Authentication finished
<@root> facebook - Logging in: Signing off..
<@root> facebook - Logging in: Connecting
<@root> facebook - Logging in: Fetching contacts
<@root> facebook - Logging in: Establishing connection
<@root> facebook - Login error: Failed to obtain SequenceID
<@root> facebook - Logging in: Signing off..
<@root> facebook - Logging in: Reconnecting in 5 seconds..
```
bitlbee 3.4.1, bitlbee-facebook f55536c2188
(will take a bit of time to get debug logs...)

**Comment by jgeboski** [comment]
Are you using a Facebook account which has no [messages](https://www.facebook.com/messages)?

**Comment by pv** [comment]
Thanks, with one message present, connection succeeded. (I usually deleted discussions afterward...)

**Comment by jgeboski** [comment]
My attention is currently diverted towards the libpurple Facebook implementation for GSoC.  So, hopefully at some point in the near future this will get fixed.

**Comment by jgeboski** [comment]
This has been resolved with 53cedfa.
